pyre/check.py

```
@@ -2607,8 +2621,12 @@ def _run_backend_bench(
                 self._record(backend, False, name, f"timeout (>{effective_timeout}s)")
                 print(f"{red('TIMEOUT')}  >{effective_timeout}s")
             else:
-                self._record(backend, False, name, f"crash (exit {code})")
-                print(f"{red('CRASH')} (exit {code})")
+                # A crash whose stderr never reaches the log is undiagnosable
+                # from a CI run, and a binary that dies before it can print
+                # anything is exactly the case worth naming.
+                detail = _first_stderr_line(stderr)
+                self._record(backend, False, name, f"crash (exit {code}){detail}")
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)